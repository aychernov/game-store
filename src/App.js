import { BrowserRouter, Routes, Route, Link } from 'react-router-dom'
import {HomePage} from "./pages/home-page";
import {Header} from "./components/header";

function App() {
  return (
      <BrowserRouter>
        <div className="App">
          <Header/>
          <Routes>
            <Route path='/' element={ <HomePage/> }/>
          </Routes>
        </div>
      </BrowserRouter>
  );
}

export default App;
