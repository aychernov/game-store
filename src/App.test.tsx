import '@testing-library/jest-dom'
import {render, screen} from "@testing-library/react";
import {App} from "./App";

describe('home-page component', () => {
	it('should render', () => {
		render(<App/>)

		expect(screen.getByText('Game Store')).toBeInTheDocument()
	});

	it('should work without data', () => {
		render(<App/>)

		expect(screen.queryByRole('props')).toBeNull()
	});

	it('should empty snapshot',  () => {
		const list = render(<App/>)
		expect(list).toMatchSnapshot()
	});

})
