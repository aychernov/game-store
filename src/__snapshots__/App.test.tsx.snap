// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home-page component should empty snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="header"
        >
          <div
            class="wrapper"
          >
            <a
              class="header__store-title"
              href="/"
            >
              Game Store
            </a>
            <a
              class="header__store-title"
              href="/about"
            >
              О магазине
            </a>
          </div>
          <div
            class="wrapper header__cart-btn-wrapper"
          />
        </div>
        <div
          class="home-page"
        >
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/forza_5.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Forza Horizon 5
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Гонки
                </span>
                <span
                  class="game-genre"
                >
                  Симулятор
                </span>
                <span
                  class="game-genre"
                >
                  Открытый мир
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    2343
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/battlefield_2042.jpg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Battlefield 2042
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Экшен
                </span>
                <span
                  class="game-genre"
                >
                  Шутер
                </span>
                <span
                  class="game-genre"
                >
                  Война
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    2433
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/life_is_strange_true_colors.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Life is Strange True Colors
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Глубокий сюжет
                </span>
                <span
                  class="game-genre"
                >
                  Протагонистка
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    3021
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/gta_v.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Grand Theft Auto V
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Открытый мир
                </span>
                <span
                  class="game-genre"
                >
                  Экшен
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    789
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/rainbow_siege.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Tom Clancy's Rainbow Six® Siege
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Тактика
                </span>
                <span
                  class="game-genre"
                >
                  Шутер
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    982
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/assassins_creed_valhalla.png);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Assassin’s Creed Valhalla
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Паркур
                </span>
                <span
                  class="game-genre"
                >
                  РПГ
                </span>
                <span
                  class="game-genre"
                >
                  Открытый мир
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    2863
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="header"
      >
        <div
          class="wrapper"
        >
          <a
            class="header__store-title"
            href="/"
          >
            Game Store
          </a>
          <a
            class="header__store-title"
            href="/about"
          >
            О магазине
          </a>
        </div>
        <div
          class="wrapper header__cart-btn-wrapper"
        />
      </div>
      <div
        class="home-page"
      >
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/forza_5.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Forza Horizon 5
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Гонки
              </span>
              <span
                class="game-genre"
              >
                Симулятор
              </span>
              <span
                class="game-genre"
              >
                Открытый мир
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  2343
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/battlefield_2042.jpg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Battlefield 2042
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Экшен
              </span>
              <span
                class="game-genre"
              >
                Шутер
              </span>
              <span
                class="game-genre"
              >
                Война
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  2433
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/life_is_strange_true_colors.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Life is Strange True Colors
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Глубокий сюжет
              </span>
              <span
                class="game-genre"
              >
                Протагонистка
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  3021
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/gta_v.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Grand Theft Auto V
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Открытый мир
              </span>
              <span
                class="game-genre"
              >
                Экшен
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  789
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/rainbow_siege.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Tom Clancy's Rainbow Six® Siege
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Тактика
              </span>
              <span
                class="game-genre"
              >
                Шутер
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  982
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/assassins_creed_valhalla.png);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Assassin’s Creed Valhalla
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Паркур
              </span>
              <span
                class="game-genre"
              >
                РПГ
              </span>
              <span
                class="game-genre"
              >
                Открытый мир
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  2863
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`home-page component should snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="header"
        >
          <div
            class="wrapper"
          >
            <a
              class="header__store-title"
              href="/"
            >
              Game Store
            </a>
            <a
              class="header__store-title"
              href="/about"
            >
              О магазине
            </a>
          </div>
          <div
            class="wrapper header__cart-btn-wrapper"
          />
        </div>
        <div
          class="home-page"
        >
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/forza_5.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Forza Horizon 5
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Гонки
                </span>
                <span
                  class="game-genre"
                >
                  Симулятор
                </span>
                <span
                  class="game-genre"
                >
                  Открытый мир
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    2343
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/battlefield_2042.jpg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Battlefield 2042
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Экшен
                </span>
                <span
                  class="game-genre"
                >
                  Шутер
                </span>
                <span
                  class="game-genre"
                >
                  Война
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    2433
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/life_is_strange_true_colors.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Life is Strange True Colors
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Глубокий сюжет
                </span>
                <span
                  class="game-genre"
                >
                  Протагонистка
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    3021
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/gta_v.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Grand Theft Auto V
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Открытый мир
                </span>
                <span
                  class="game-genre"
                >
                  Экшен
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    789
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/rainbow_siege.jpeg);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Tom Clancy's Rainbow Six® Siege
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Тактика
                </span>
                <span
                  class="game-genre"
                >
                  Шутер
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    982
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="game-item"
          >
            <div
              class="game-cover"
              style="background-image: url(/game-covers/assassins_creed_valhalla.png);"
            />
            <div
              class="game-item__details"
            >
              <span
                class="game-item__title"
              >
                Assassin’s Creed Valhalla
              </span>
              <div
                class="game-item__genre"
              >
                <span
                  class="game-genre"
                >
                  Паркур
                </span>
                <span
                  class="game-genre"
                >
                  РПГ
                </span>
                <span
                  class="game-genre"
                >
                  Открытый мир
                </span>
              </div>
              <div
                class="game-item__buy"
              >
                <div
                  class="game-buy"
                >
                  <span
                    class="game-buy__price"
                  >
                    2863
                     руб.
                  </span>
                  <a
                    class="login-link"
                    href="/login"
                  >
                     
                    <br />
                     Авторизуйтесь для покупок! :P
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="header"
      >
        <div
          class="wrapper"
        >
          <a
            class="header__store-title"
            href="/"
          >
            Game Store
          </a>
          <a
            class="header__store-title"
            href="/about"
          >
            О магазине
          </a>
        </div>
        <div
          class="wrapper header__cart-btn-wrapper"
        />
      </div>
      <div
        class="home-page"
      >
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/forza_5.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Forza Horizon 5
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Гонки
              </span>
              <span
                class="game-genre"
              >
                Симулятор
              </span>
              <span
                class="game-genre"
              >
                Открытый мир
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  2343
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/battlefield_2042.jpg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Battlefield 2042
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Экшен
              </span>
              <span
                class="game-genre"
              >
                Шутер
              </span>
              <span
                class="game-genre"
              >
                Война
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  2433
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/life_is_strange_true_colors.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Life is Strange True Colors
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Глубокий сюжет
              </span>
              <span
                class="game-genre"
              >
                Протагонистка
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  3021
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/gta_v.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Grand Theft Auto V
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Открытый мир
              </span>
              <span
                class="game-genre"
              >
                Экшен
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  789
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/rainbow_siege.jpeg);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Tom Clancy's Rainbow Six® Siege
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Тактика
              </span>
              <span
                class="game-genre"
              >
                Шутер
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  982
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="game-item"
        >
          <div
            class="game-cover"
            style="background-image: url(/game-covers/assassins_creed_valhalla.png);"
          />
          <div
            class="game-item__details"
          >
            <span
              class="game-item__title"
            >
              Assassin’s Creed Valhalla
            </span>
            <div
              class="game-item__genre"
            >
              <span
                class="game-genre"
              >
                Паркур
              </span>
              <span
                class="game-genre"
              >
                РПГ
              </span>
              <span
                class="game-genre"
              >
                Открытый мир
              </span>
            </div>
            <div
              class="game-item__buy"
            >
              <div
                class="game-buy"
              >
                <span
                  class="game-buy__price"
                >
                  2863
                   руб.
                </span>
                <a
                  class="login-link"
                  href="/login"
                >
                   
                  <br />
                   Авторизуйтесь для покупок! :P
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
